packages/relay-compiler/language/RelayLanguagePluginInterface.js

```diff
@@ -36,6 +36,7 @@ export type PluginInterface = {
   findGraphQLTags: GraphQLTagFinder,
   formatModule: FormatModule,
   typeGenerator: TypeGenerator,
+  schemaExtensions?: $ReadOnlyArray<string>,
```

Comment:
I'm not sure this should be in the PluginInterface.
Can we move it to the compiler configs, instead?

We have this `type Config {...}` in the RelayCompilerMain.js - maybe we could add another option there, next to the `customScalars?: ScalarTypeMapping`.

And then you can provide those directive extensions via `relay-config`, right?

Reply:
Thanks for the review!

Hmm, I'd actually like to argue that it belongs in `PluginInterface` or something exposed by the language plugin. This because it (or at least the particular thing I'm trying to do) is deeply integrated with the actual language plugin (and only makes sense in the language plugin context). Adding it to `relay-config` would mean that every user would have to configure it themselves in addition to using the language plugin, or am I missing something?

Reply:
Oh, I guess you're right! I didn't think about this particular use-case.